webapp/src/index.tsx

```diff
@@ -72,6 +72,18 @@ export default class Plugin {
                         icon: 'fa-users', // font-awesome-4.7.0 handler
                         value: siteStats?.total_connected_users || 0,
                     },
+                    msteams_invited_users: {
+                        name: 'MS Teams: Invited Users',
+                        id: 'msteams_invited_users',
+                        icon: 'fa-users', // font-awesome-4.7.0 handler
+                        value: siteStats?.pending_invited_users || 0,
+                    },
+                    msteams_whitelisted_users: {
+                        name: 'MS Teams: Whitelisted Users',
+                        id: 'msteams_whitelisted_users',
+                        icon: 'fa-users', // font-awesome-4.7.0 handler
+                        value: siteStats?.current_whitelist_users || 0,
+                    },
```

Comment:
I'm wondering if we should consider keeping this as a metric / Grafana dashboard only for now, vs. exposing to the enduser? It's the kind of data that's transiently useful (vs. connected count, which shows volume / adoption).

Reply:
I'm perfectly happy to remove them, but I do think they provide some value in monitoring or any potential troubleshooting during the initial rollout. 1/5

Reply:
We could leave them as "extra" stats in the api request, but then just not show them in the site statistics page.